diff --git a/aamp/CMakeLists.txt b/aamp/CMakeLists.txt
index 03323669..96b77eda 100644
--- a/aamp/CMakeLists.txt
+++ b/aamp/CMakeLists.txt
@@ -125,7 +125,7 @@ else()
 	message("CMAKE_WPEWEBKIT_JSBINDINGS and CMAKE_WPEWEBKIT_JSBINDINGS not set")
 endif()
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-multichar -std=c++11 -Wno-non-virtual-dtor -Werror=effc++")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-multichar -std=c++11 -Wno-non-virtual-dtor ")
 
 if(CMAKE_IARM_MGR)
 	message("CMAKE_IARM_MGR set")
